### PR TITLE
dev-util/wstool: EAPI 7, install completions in the right paths

### PR DESCRIPTION
**Pull request by Alessandro-Barbieri**
Closes: https://bugs.gentoo.org/693248
Package-Manager: Portage-2.3.77, Repoman-2.3.17
Signed-off-by: Alessandro Barbieri <lssndrbarbieri@gmail.com>

### CODE REVIEW

**Comment by Alessandro-Barbieri** [comment]
@aballier could you review this?

File: dev-util/wstool/wstool-9999.ebuild [hunk 1]
```diff
@@ -43,10 +42,20 @@ DEPEND="${RDEPEND}
 		dev-vcs/subversion
 	)
 "
+BDEPEND="dev-python/sphinx"
 
-PATCHES=( "${FILESDIR}/prefix2.patch" )
```
Comment (aballier) [inline dev-util/wstool/wstool-9999.ebuild]: you are dropping a patch here

File: dev-util/wstool/wstool-9999.ebuild [hunk 2]
```diff
@@ -43,10 +42,20 @@ DEPEND="${RDEPEND}
...
+PATCHES=( "${FILESDIR}/do-not-install-completion.patch" )
 DISTUTILS_IN_SOURCE_BUILD="yes"
 
+python_install_all() {
+	distutils-r1_python_install_all
+
+	newbashcomp completion/wstool-completion.bash "${PN}"
+	insinto /usr/share/zsh/site-functions
+	doins completion/wstool-completion.bash
```
Comment (aballier) [inline dev-util/wstool/wstool-9999.ebuild]: a .bash for zsh does not look right

Comment (Alessandro-Barbieri) [reply dev-util/wstool/wstool-9999.ebuild]: It's upstream fault

File: dev-util/wstool/wstool-9999.ebuild (hunk 1 above)
Comment (Alessandro-Barbieri) [reply dev-util/wstool/wstool-9999.ebuild]: I'm substituting the patch

**Comment by Alessandro-Barbieri** [comment]
rebasing

File: dev-util/wstool/wstool-9999.ebuild (hunk 2 above)
Comment (Alessandro-Barbieri) [reply dev-util/wstool/wstool-9999.ebuild]: What should I do with this?

**Comment by juippis** [comment]
45eb8e78e207e04ce46f9ce9975452e4063d5b14

**Pull request closed without merging**